raaghu-components/src/rds-comp-chat/rds-comp-chat.tsx

```
@@ -140,36 +149,34 @@ const RdsCompUserComments = (props: RdsCompUserCommentsProps) => {
     setShowEmojiPicker(false); // Close the emoji picker
   };
 
-  const handleDeleteComment = (index: number) => {
-  if (!allowDelete) return;
-
-  // Get the comment to be deleted
-  const commentToDelete = commentList[index];
-  const commentId = commentToDelete?.CommentId;
+  const handleDeleteComment = (commentId: number, tempId: string) => {
```

ESLint

[error] Expected indentation of 4 spaces but found 2.

(indent)